src/app/profiles/profiles.default.ts

```diff
@@ -0,0 +1,116 @@
+// --------- BEGIN RUNBOX LICENSE ---------
+// Copyright (C) 2016-2018 Runbox Solutions AS (runbox.com).
+//
+// This file is part of Runbox 7.
+//
+// Runbox 7 is free software: You can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+//
+// Runbox 7 is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Runbox 7. If not, see <https://www.gnu.org/licenses/>.
+// ---------- END RUNBOX LICENSE ----------
+
+import { Component } from '@angular/core';
+import { RMM } from '../rmm';
+import { RunboxWebmailAPI } from '../rmmapi/rbwebmail';
+import { timeout, share, retry } from 'rxjs/operators';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+    selector: 'app-profiles-default',
+    styleUrls: ['profiles.default.scss'],
+    templateUrl: 'profiles.default.html',
+})
+export class DefaultProfileComponent {
+    field_errors: any;
```

Comment:
This looks to be unused (apart from being passed to the rmm methods), perhaps it should get displayed somewhere in profiles.default.html?